op-devstack/presets/minimal/advance_test.go

```diff
@@ -0,0 +1,31 @@
+package minimal
+
+import (
+	"testing"
+	"time"
+
+	"github.com/ethereum-optimism/optimism/op-devstack/devtest"
+	"github.com/ethereum-optimism/optimism/op-devstack/presets"
+	"github.com/stretchr/testify/require"
+)
+
+func TestCLAdvance(gt *testing.T) {
```

Comment:
Thank you for the test. I validated that the test is passing, with using the minimal preset. Since this will be the basic acceptance tests, can we move this to the op-acceptance-tests/tests/base/basic, as well as other tests?